### ISSUE TITLE
the datasets of Fine-Tuning network?which?

### ISSUE THREAD

**Issue by henbucuoshanghai** [showlab/Tune-A-Video#91]
the fine tuning process,

**Comment by henbucuoshanghai** [comment]
how to train it? the input and outputs of finetune network？inputs are txt and video?oututs is the same video?
the Reconstruction Loss is?
